src/app/constants.ts

```diff
@@ -16,6 +16,10 @@ export function isValidOrcidFormat(id) {
   return id && regExp.test(id)
 }
 
+export function getOrcidNumber(orcid) {
+  return orcid.match(ORCID_REGEXP)[0]
```

Comment:
@DanielPalafox should this be using the `ORCID_URI_REGEXP`? instead of the ORCID_REGEXP

Reply:
@amontenegro No, necessarily. You can use both expressions to achieve this but I am using ORCID_REGEXP because it will look specifically for the Orcid id and in the other part the ORCID_URI_REGEXP it is to validate the domain and the Orcid id